parsing/src/processor.ts

```diff
@@ -93,6 +93,64 @@ function getItems(data: any[]): PartDataInterface {
         .filter((entry: any) => entry.Classes)
         .flatMap((entry: any) => entry.Classes)
         .forEach((entry: any) => {
+            
+            //These are exception products that aren't produced by mines or extractors, they are raw materials
+            if (entry.ClassName === "Desc_Leaves_C") {
```

Comment:
Rather than doing a load of conditionals within a loop, can't we just add them in verbatim after the forEach? Sure it won't be in order but we can just do a sort at the end if we really wanted to.

This way these multiple if statements are not being executed for **every** recipe in the game.

Reply:
Definitely focused on a 'build, then optimize' methodology. The code runs in less than a second. Now that it works, I want to focus on tests and optimizing. In Particular, I think some of these if's can be removed with some tests and optimization.